Docs/Workflows.md

```diff
@@ -99,3 +99,26 @@ jobs:
 If this is for a submodule of [Lombiq's Open-Source Orchard Core Extensions](https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/), the `repo` input can be omitted, because the above is its default value. Otherwise, use your parent repository's address in the `{owner}/{repo_name}` format.
 
 Refer to [Github Actions reusable workflows](https://docs.github.com/en/actions/learn-github-actions/reusing-workflows#overview) for more information.
+
+## Jira issue creation for community activities workflow
+
+Creates Jira issues for community activities happening on GitHub, like issues, discussions, and pull requests being opened. Pull requests are only taken into account if they're not already related to a Jira issue (by starting their title with a Jira issue key).
+
+Set up secrets for the `JIRA_*` parameters as explained [here](https://github.com/marketplace/actions/jira-login#enviroment-variables). You may use secret names without the `DEFAULT_*` prefix, but that's our recommendation for organization-level secrets, so you have defaults but can override them on a per-repository basis.
+
+The secrets with the `*_JIRA_ISSUE_DESCRIPTION` suffix should contain templates for the Jira issues to be created, using the internal markup format of Jira (not Markdown). Example for one for `ISSUE_JIRA_ISSUE_DESCRIPTION`:
+
+```text
+h1. Summary
+See the linked GitHub issue, including all the comments.
+
+h1. Checklist
+* Assign yourself to the referenced GitHub issue.
+* [Issue completion checklist|https://example.com/checklist]
+```
+
+All three templates are optional and if not provided, defaults will be used.
```

Comment:
[The workflow inputs are still required though.](https://github.com/Lombiq/GitHub-Actions/pull/50/files#diff-724e9168a79d65edb0fe14c60f55080e9e2f8a5090e654273248f4e9e9156ebbR15-R20)

Reply:
I'm not done yet ;).

Reply:
Sorry.